.github/CONTRIBUTING.md

```diff
@@ -41,6 +41,12 @@ Note: It may be beneficial to create a draft PR while working on your command. T
 
 You can find the pull request template [here](PULL_REQUEST_TEMPLATE.md).
 
+## Setting up the env
+
+1. Copy the `.env.example` file and rename it to `.env`.
+2. In your `.env` file replace `NODE_CONFIG_ENV=staging` with `NODE_CONFIG_ENV=development`.
+3. Update the `TOKEN` with your own if you wish to use certain features.
```

Comment:
I assume this, refers to updating TOKEN variables required for 3rd parties like the AVWX, WolframAlpha, etc.? If that's the case I would put it into a separate note under the `Setting up the bot` header.
```suggestion
```